trashtalk/utils.py

```diff
@@ -48,34 +46,30 @@ def get_location(address):
     :param city: `str`, Defaults to Oakland
     :return: `namedtuple`, `Place` with full location data
     """
-    location=google.geocode(address)
-    # print(location.address)
-    # first_split=location.address.split(",")
-    # street_split=first_split[0].split(" ")
-    # zipcode_split=first_split[2].split(" ")
-
-    return Place(address=location.address, latitude=location.latitude,
-                 longitude=location.longitude)
-
-
-    # try:
-    #     res = geolocator.geocode(' '.join(address))
-    # except (GeocoderQueryError, GeopyError):
-    #     app.logger.exception("Geopy error.")
-    #     raise
-    # else:
-    #     app.logger.info("Geolocator successful!: %s", address)
-    #     print(res.address)
-    #     location = res.address.split(',')
-    #     if len(location) == 7:
-    #         # Locations should only be of length 7 or 9
-    #         # Handle for missing name and building number
-    #         location.insert(0, None) * 2
-    #     app.logger.info("Location: %s", location)
-    #     return Place(name=location[0], number=location[1], street=location[2],
-    #                  district=location[3], city=location[4], county=location[5],
-    #                  state=location[6], zipcode=location[7], country=location[8],
-    #                  latitude=res.latitude, longitude=res.longitude)
+    try:
+        res = geolocator.geocode(address)
```

Comment:
Why choose **geolocator** over **google** for this function?

Reply:
Because the geolocator was the most practical first step for what we need. The Google module for geopy is for geolocation. Geolocation converts *addresses* to *coordinates*. Thus if a user has an address, no need for coordinates. The google maps api serves that purpose just fine. Cross street I think is a more valuable use case to work on right now.

That said, geolocation is on the radar but I thought it best to start simpler first and add features as we go.